nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -893,31 +886,42 @@ define([
             const events = Events.make(),
                 content = div(
                     {
-                        class: 'kbase-extension kb-app-cell kb-app-cell__container',
+                        class: `kbase-extension ${cssCellType} ${cssCellType}__container`,
                     },
                     [
                         div(
                             {
-                                class: 'kb-app-cell__prompt prompt',
+                                class: `${cssCellType}__prompt prompt`,
                                 dataElement: 'prompt',
                             },
-                            [div({ dataElement: 'status' })]
+                            [
+                                div({
+                                    class: `${cssCellType}__prompt_status`,
+                                    dataElement: 'status',
+                                }),
+                            ]
                         ),
                         div(
                             {
-                                class: 'kb-app-cell__body body',
+                                class: `${cssCellType}__body body`,
                                 dataElement: 'body',
                             },
                             [
                                 div(
                                     {
-                                        class: 'kb-app-cell__widget_container',
+                                        class: `${cssCellType}__widget_container`,
                                         dataElement: 'widget',
                                     },
                                     [
                                         div({ class: 'container-fluid' }, [
                                             buildRunControlPanel(events),
                                         ]),
+                                        div(
```

Comment:
moved the tab-pane + widget to mirror the structure in the bulk import cell. This means changing the path used to access the tab-pane from `run-control-panel.tab-pane` to `body.widget.tab-pane`.